gdeploylib/__init__.py

```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 
+from global_vars import Global
```

Comment:
Ignore this. I was observing how `import *' behaves on the ordering of the import statements.